students/preactive/Lesson3/list_lab.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
+
+def series_one():
+    # Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+    # Display the list (plain old print() is fine…).
+    fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+    print(fruits)
+
+    # Ask the user for another fruit and add it to the end of the list.
+    # Display the list.
+    response_one = input("Add a fruit:\n>")
+    fruits.append(response_one)
+    print(fruits)
+
+    # Ask the user for a number and display the number back to the user and
+    # the fruit corresponding to that number (on a 1-is-first basis).
+    # Remember that Python uses zero-based indexing,
+    # so you will need to correct.
+    num_fruits = len(fruits)
+    response_two_txt = "Pick a integer from 1 to %s:\n>" % str(num_fruits)
+    response_two = input(response_two_txt)
+    try:
+        response_two_int = int(response_two)
+        if response_two_int not in range(1, num_fruits + 1):
+            print("Not in range. Try again, mouth breather.")
+            series_one()
+        else:
+            print(response_two + ": " + str(fruits[response_two_int - 1]))
+    except:
+        print("Not in ranges. Try again, mouth breather.")
+        series_one()
+
+    # Add another fruit to the beginning of the
+    # list using “+” and display the list.
+    fruits = ["tomato"] + fruits
+    print(fruits)
+
+    # Add another fruit to the beginning of the list
+    # using insert() and display the list.
+    fruits.insert(len(fruits), "banana")
+    print(fruits)
+
+    # Display all the fruits that begin with “P”, using a for loop.
+    for fruit in fruits:
+        try:
+            if fruit[0:1].lower() == "p":
+                print(fruit)
+        except:
+            print("Something went wrong with lower method. "
+                  "But did you know you could save 15% or more "
+                  "on your car insurance by switching to Geiko?")
+    return fruits
+
+
+def series_two():
+    series_one_stub = series_one()
+
+    # Display the list.
+    print(series_one_stub)
+
+    # Remove the last fruit from the list.
+    series_one_stub.pop()
+
+    # Display the list.
+    print(series_one_stub)
+
+    # Ask the user for a fruit to delete, find it and delete it.
+    delete_fruit = input("enter a fruit to remove. CaSe SeNsItIvE:\n>")
```

Comment:
You could use the `lower()` method here to make case insensitive

Reply:
I like to remind people of case, as the majority of the internet runs off of *nix of some kind.  The only places where I like to normalize is login IDs as this causes problems when people login.  that one time when register for a service and you use "Preactive" and then 3 years later you login with preactive and it says your not a user.  [triggered].  until some decent admin picks up the ticket and looks at the database for another user of the same case and unlocks it.